### ISSUE TITLE
Pyinstaller and Alive-Progress don't like eachother.

### ISSUE THREAD

**Issue by MrKelpy** [rsalmei/alive-progress#149]
I've recently made a small little app using alive-progress, and when I used pyinstaller to make an executable out of it, the executable broke every single time I tried to launch it. 
`> File "main.py", line 33, in <module>

  File "ModpackDownloader.py", line 50, in start

  File "ModpackDownloader.py", line 90, in _download_modpack

  File "alive_progress\core\progress.py", line 106, in alive_bar

  File "alive_progress\core\configuration.py", line 149, in create_context

  File "alive_progress\core\configuration.py", line 183, in lazy_init

  File "alive_progress\core\configuration.py", line 14, in _spinner_input_factory

  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked

  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module

  File "alive_progress\styles\__init__.py", line 1, in <module>

  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked

  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module

  File "alive_progress\styles\exhibit.py", line 9, in <module>

  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked

  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module

  File "alive_progress\styles\internal.py", line 126, in <module>

  File "alive_progress\styles\internal.py", line 12, in __create_spinners

  File "alive_progress\animations\spinners.py", line 43, in frame_spinner_factory

  File "alive_progress\animations\spinners.py", line 43, in <genexpr>

  File "alive_progress\animations\spinners.py", line 43, in <genexpr>

  File "alive_progress\utils\cells.py", line 145, in to_cells

  File "alive_progress\utils\cells.py", line 149, in split_graphemes

  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked

  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module

  File "grapheme\__init__.py", line 9, in <module>

  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked

  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module

  File "grapheme\api.py", line 2, in <module>

  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked

  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module

  File "grapheme\finder.py", line 3, in <module>

  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked

  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module

  File "grapheme\grapheme_property_group.py", line 97, in <module>

FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\Alex\\AppData\\Local\\Temp\\_MEI164842\\grapheme\\data/grapheme_break_property.json'`

This was the error that popped up everytime I used it. Is there anyway to fix it?
I'm on Windows 10, Python 3.9.

**Comment by TheTechRobo** [comment]
Duplicate of #123?

**Comment by rsalmei** [comment]
Yes... please look for closed issues before creating a new one, will you? 👍
